CONTRIBUTORS.yaml

```
@@ -288,6 +293,11 @@ brigidagallone:
     orcid: 0000-0003-3045-5949
     joined: 2022-05
 
+burkemlou:
+    name: Melissa Burke
+    email: melissa@biocommons.org.au
+    joined: 2023-12
```

```suggestion
    joined: 2023-12
    affiliations:
    - Australian-BioCommons
```

is how affiliations can be added